templates/css/product.css

```diff
@@ -65,8 +78,8 @@ amp-carousel {
   position:absolute;
   top:10px;
```

Comment:
Would it look OK to change this to space-1 or space-2?

Reply:
done